src/components/modules/Chapter/index.tsx

```diff
@@ -32,26 +36,29 @@ const Chapter: React.FC<IChapterProps> = ({ data, pageContext }) => {
   }
 
   // TODO: memoize this:
-  const arabicChapter = data.api.chapter.translations.find(({ locale_code }) => locale_code === 'ar')
+  const arabicChapter = chapterTranslation.find(({ locale_code }) => locale_code === 'ar')
 
   const arabicTranscription = arabicChapter && arabicChapter.transcription
-  const chapterStrings = data.api.chapter.translations.find(({ locale_code }) => locale_code === pageContext.locale)
+  const chapterStrings = chapterTranslation.find(({ locale_code }) => locale_code === pageContext?.locale)
 
   if (!chapterStrings) {
     // TODO: Log to Sentry
     return <div>{"Error, we didn't find any strings for this chapter ..."}</div>
   }
-  const otherLocalePath = `${data.otherLocaleTranslations.localePath}${data.api.chapter.course.track.slug}/${data.api.chapter.course.slug}/${data.api.chapter.slug}`
+  const otherLocalePath = `${data?.otherLocaleTranslations.localePath}${data?.api.chapter.course.track.slug}/${data?.api.chapter.course.slug}/${data?.api.chapter.slug}`
 
-  const navigationButtons = (
+  const navigationButtons = t && pageContext && (
```

Comment:
this condition is a bit strange

Reply:
This was to fix ts linting, it was broke in this file